app-pda/usbmuxd/usbmuxd-1.1.1-r1.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools systemd udev
+
+DESCRIPTION="USB multiplex daemon for use with Apple iPhone/iPod Touch devices"
+HOMEPAGE="https://libimobiledevice.org/"
+SRC_URI="https://github.com/libimobiledevice/usbmuxd/releases/download/${PV}/${P}.tar.bz2"
+
+# src/utils.h is LGPL-2.1+, rest is found in COPYING*
+LICENSE="GPL-2 GPL-3 LGPL-2.1+"
+SLOT="0"
+KEYWORDS="amd64 ~arm ~ppc ~ppc64 x86"
+IUSE="selinux systemd"
+
+DEPEND="
+	acct-user/usbmux
+	>=app-pda/libimobiledevice-1.3.0:=
+	>=app-pda/libplist-2.2:=
+	virtual/libusb:1=
+"
+
+RDEPEND="
+	${DEPEND}
+	virtual/udev
+	selinux? ( sec-policy/selinux-usbmuxd )
+	systemd? ( sys-apps/systemd )
+"
+
+BDEPEND="
+	virtual/pkgconfig
+"
+
+pkg_postrm() {
+	udev_reload
+}
+
+pkg_postinst() {
+	udev_reload
+}
+
+src_prepare() {
+	default
+	eautoreconf
```

Comment:
Do we actually need the eautoreconf here? We're not applying any patches to the build system and we seem to be downloading a release tarball.